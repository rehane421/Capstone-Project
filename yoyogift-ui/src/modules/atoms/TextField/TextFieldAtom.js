import React from "react";
import TextField from "@material-ui/core/TextField";

export const TextFieldAtom = ({
  id,
  label,
  name,
  autoComplete,
  onChange,
  setEmail,
  setPassword
}) => {
  return (
    <div>
      <TextField
        variant="outlined"
        margin="normal"
        required
        fullWidth
        id={id}
        label={label}
        name={name}
        autoComplete={name}
        autoFocus
        onChange={onChange}
      />
    </div>
  );
};
export default TextFieldAtom;
