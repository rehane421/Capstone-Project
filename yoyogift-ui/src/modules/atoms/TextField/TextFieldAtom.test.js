import React from "react";
import { shallow } from "../../../enzyme";
import { shallowToJson } from "enzyme-to-json";
import { TextFieldAtom } from "./TextFieldAtom";

let wrapper;
it("should render TextFieldAtom component", () => {
  wrapper = shallow(<TextFieldAtom />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
