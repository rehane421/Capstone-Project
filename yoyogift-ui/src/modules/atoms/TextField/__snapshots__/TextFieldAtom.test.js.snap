// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TextFieldAtom component 1`] = `
<div>
  <WithStyles(ForwardRef(TextField))
    autoFocus={true}
    fullWidth={true}
    margin="normal"
    required={true}
    variant="outlined"
  />
</div>
`;
